docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -0,0 +1,263 @@
+      chipRef.current.blur();
+    }
+  };
+  return (
+    <Chip
+      label="Chip Outlined"
+      variant="outlined"
+      ref={chipRef}
+      onKeyUp={keyUpHandler}
+    />
+  );
+}
+```
+
+### Grid v2 (Unstable_Grid)
+
+The spacing mechanism was reworked to use the `gap` CSS property.
+This maps better with current layout practices and removes the need for using React Context.
+It brings some breaking changes described in the following sections.
+
+#### The Grid is contained inside parent's padding
+
+Previously, the Grid overflowed its parent.
+In v6, this is fixed:
+
+<img src="/static/material-ui/migration-v5/grid-overflow-change.png" style="width: 814px;" alt="Before and after of the Grid no longer overflowing its parent in v6." width="1628" height="400" />
+
+This removes the need for the `disableEqualOverflow` prop, so you can remove it:
+
+```diff
+- <Grid disableEqualOverflow />
++ <Grid />
+```
+
+#### Spacing is no longer considered inside the Grid item's box
+
+Previously, the Grid items included spacing in it's box.
+In v6, this is fixed:
+
+<img src="/static/material-ui/migration-v5/grid-spacing-change.png" style="width: 814px;" alt="Before and after of the Grid items no longer including spacing in their box." width="1628" height="400" />
+
+:::warning
+Both of these changes might slightly affect your layout.
+Note that the items' position doesn't change.
+We recommend adopting this new behavior and **not trying to replicate the old one**, as this is a more predictable and modern approach.
+:::
+
+## Stabilized APIs
+
+### CssVarsProvider and extendTheme
+
+The `CssVarsProvider` and `extendTheme` APIs are now stable.
+If you already use them in v5, you can now drop the experimental prefix.
+
+```diff
+- import { experimental_extendTheme as extendTheme, Experimental_CssVarsProvider as CssVarsProvider } from '@mui/material/styles';
++ import { extendTheme, CssVarsProvider } from '@mui/material/styles';
+```
+
+Check out the [CSS theme variables page](/material-ui/customization/css-theme-variables/overview/) to learn more about it.
+
+### Specific mode styles
+
+The `theme.applyStyles` is a new utility for creating style for a specific mode.
+It's designed to replace the use of `theme.palette.mode` when applying light or dark styles.
+
+```diff
+ const MyComponent = styled('button')(({ theme }) => ({
+   padding: '0.5rem 1rem',
+   border: '1px solid,
+-  borderColor: theme.palette.mode === 'dark' ? '#fff' : '#000',
++  borderColor: '#000',
++  ...theme.applyStyles('dark', {
++    borderColor: '#fff',
++  })
+ }))
+```
+
+Use these codemods to migrate your project to `theme.applyStyles`:
+
+```bash
+npx @mui/codemod@next v6.0.0/styled <path/to/folder-or-file>
+npx @mui/codemod@next v6.0.0/sx-prop <path/to/folder-or-file>
+npx @mui/codemod@next v6.0.0/theme-v6 <path/to/theme-file>
+```
+
+> Run `v6.0.0/theme-v6` against the file that contains the custom `styleOverrides`. Ignore this codemod if you don't have a custom theme.
```

Comment:
```suggestion
:::info
Run `v6.0.0/theme-v6` against the file that contains the custom `styleOverrides`. Ignore this codemod if you don't have a custom theme.
:::
```

Use callout instead of blockquote